requirements.txt

```diff
@@ -0,0 +1,3 @@
+coverage
```

Comment:
no specific version?

Reply:
Addressed in 72ed5726450617cb5661ad95cca3138df26f2a8d.